invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/container.overrides

```diff
@@ -81,3 +81,12 @@
 .ui.container.requests-feed-container {
   position: relative;
 }
+
+.timeline-comment-editor-container .ui.fluid.container {
```

Comment:
I have a feeling I saw these attributes implemented for some other components, if yes, can we make it centralized?